### DISCUSSION TITLE
savedView with RealityData

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#6032] [Q&A]
Hi everyone.
How can I activate any reality data connected to the view through the save views.
My client complains that when he activates saved views, reality data is not displayed
From the json returned from saveviews in "itwin3dView" "displayStyle"
  I have contextRealityModels where there are reality data attached to the savedview but the "DisplayStyle3dSettingsProps" interface seems not to implement them, but more likely I don't do it

thanks Paul

**Comment by pmconne** [comment]
Look at the network requests in the browser's development tools while applying the saved view.
You should see requests being made to try to obtain the reality data's tileset(s).
Most likely, those are failing, and the response codes can tell you why.
Quite likely, the response is a 401 indicating the user is not authorized to access the reality data, either because they are not signed in, or they were never given access to it.

**Comment by Paolo-lab** [comment]
Hi thanks for the reply.
Unfortunately I don't have any network calls to Reality Data. 
this is the static method I use to load saved views.
and in savedview I have my reality data as per the attached image

<img width="657" alt="Reality1" src="https://github.com/iTwin/itwinjs-core/assets/57446652/e84af9e2-ee0a-4353-aad3-1b97f7f64868">

as you can see, inside savedView I have my reality shows which are 8 I think, but when I do viewState.setDisplayStyle(viewPort.displayStyle) inside viewport I don't have any reality shows loaded.


<img width="848" alt="reality0" src="https://github.com/iTwin/itwinjs-core/assets/57446652/0532eda4-ca8b-4775-9956-abc88ca323d6">

Thanks to you, I was able to load a saved view at the launch of the model through the legacyView #5983 and having also implemented the reality part, I can see both the saved view and the attached reality shows.
should I perhaps use legacy or what am I doing wrong?

tanks Paul
